Packs/DSPM/Integrations/DSPM/README.md

```diff
@@ -0,0 +1,946 @@
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Alert Status Update
+>| Alert ID | Old Status | New Status |
+>|---|---|---|
+>| 000000608 | INVESTIGATING | INVESTIGATING |
+
+### dspm-list-alerts
+
+***
+Fetch list of alerts.
+
+#### Base Command
+
+`dspm-list-alerts`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| detection_time_equals | Exact detection time (equals). detection time format - YYYY-MM-DDTHH:MM:SSZ. | Optional | 
+| detection_time_greater_than_or_equal | Detection time (greater than or equal). detection time format - YYYY-MM-DDTHH:MM:SSZ. | Optional | 
+| detection_time_greater_than | Detection time (greater than). detection time format - YYYY-MM-DDTHH:MM:SSZ. | Optional | 
+| detection_time_less_than_or_equal | Detection time (less than or equal). detection time format - YYYY-MM-DDTHH:MM:SSZ. | Optional | 
+| detection_time_less_than | Detection time (less than). detection time format - YYYY-MM-DDTHH:MM:SSZ. | Optional | 
+| policy_name_in | A comma-separated list of policy names. | Optional | 
+| policy_name_equals | Exact policy name. | Optional | 
+| asset_name_in | A comma-separated list of asset names. | Optional | 
+| asset_name_equals | Exact asset name. | Optional | 
+| cloud_provider_in | A comma-separated list of cloud providers. Possible values are: AWS, AZURE, GCP, SNOWFLAKE, FILE_SHARE, O365. | Optional | 
+| cloud_provider_equals | Exact cloud provider. Possible values are: AWS, AZURE, GCP, SNOWFLAKE, FILE_SHARE, O365. | Optional | 
+| destination_project_vendor_name_in | A comma-separated list of project vendor names. | Optional | 
+| destination_project_vendor_name_equals | Exact destination project vendor name. | Optional | 
+| cloud_environment_in | A comma-separated list of cloud environments. Possible values are: UNKNOWN, DEVELOPMENT, STAGING, TESTING, PRODUCTION. | Optional | 
+| cloud_environment_equals | Exact cloud environment. Possible values are: UNKNOWN, DEVELOPMENT, STAGING, TESTING, PRODUCTION. | Optional | 
+| policy_severity_in | A comma-separated list of policy severities. Possible values are: HIGH, MEDIUM, LOW. | Optional | 
+| policy_severity_equals | Exact policy severity. Possible values are: HIGH, MEDIUM, LOW. | Optional | 
+| category_type_in | A comma-separated list of category types. Possible values are: FIRST_MOVE, ATTACK, COMPLIANCE, ASSET_AT_RISK, RECONNAISSANCE. | Optional | 
+| category_type_equals | Exact category type. Possible values are: FIRST_MOVE, ATTACK, COMPLIANCE, ASSET_AT_RISK, RECONNAISSANCE. | Optional | 
+| status_in | A comma-separated list of statuses. Possible values are: OPEN, CLOSED, UNIMPORTANT, WRONG, HANDLED, INVESTIGATING. | Optional | 
+| status_equals | Exact status. Possible values are: OPEN, CLOSED, UNIMPORTANT, WRONG, HANDLED, INVESTIGATING. | Optional | 
+| sort | Sort order (property,asc\|desc). | Optional | 
+| limit | The maximum number of alerts to retrieve. Default is 50. | Optional | 
+
```

Comment:
For the above arguments, lets add "The" at the start of each description, apart from where we have "A comma-separated list.." For example, "The exact category type."